nbbuild/build.xml

```diff
@@ -1967,7 +1967,6 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="diff/test/unit/data/**" /> <!--test data-->
                 <exclude name="diff/test/unit/src/org/netbeans/modules/diff/builtin/provider/*.txt" /> <!--test data-->
                 <exclude name="diff/test/unit/src/org/netbeans/modules/diff/builtin/visualizer/data/**" /> <!--test data-->
-                <exclude name="javafx2.project/src/org/netbeans/modules/javafx2/project/templates/**" /> <!--license would be visible when users edit the templates inside their IDE-->
```

Comment:
Could you please re-do this merge yourself?

I tried to do it with the GitHub editor and now I deleted this line because GitHub had an older repository state in mind somehow...